src/components/axes/timeAxis.ts

```diff
@@ -278,14 +285,40 @@ export module Axis {
       return Math.min(this._getIntervalLength(config), this.width()) >= worstWidth;
     }
 
+    /**
+     * Makes sure that the size it requires is a multiple of tier sizes, such that
+     * we have no leftover tiers
+     */
```

Comment:
This comment is leaking into the plottable.d.ts; can you move it inside the implementation?